CanalTP/Chaos
# History disruption step 1

# Description

This add the route /disruptions/{UIID}/history and return a versionnning list of the disruption (just the modification of the disruption -> not the impacts) 

## Issue

Issue link: BOT-1311

## How to test

Here are the following steps to test this pull request:

- Create then change the disruption properties (multi-time)
- Call the API with /disruptions/{UIID}/history
- You should see all modification except for the impacts parts

## AT

|Given|When|Then|
|---|---|---|
|J'ai créé une perturbation|Je consulte l'historique de cette perturbation|J'ai une liste avec 1 seule entrée, l'état actuel de la perturbation et sa date de création|
|Je modifie la cause de cette perturbation|Je consulte l'historique de cette perturbation|J'ai une liste avec 2 entrées, la première entrée est l'état de création, et la deuxième entrée est l'état actuel avec leur date de modification (Chaque entrée correspond à une perturbation complète)|
|J'ai la même perturbation où je ne change rien mais je "put" dans la base|Je consulte l'historique de cette perturbation|J'ai une liste avec 3 entrées (changement de comportement à prévoir)|
|J'ai des perturbations qui existent avant le développement de cette feature|Je consulte l'historique de cette perturbation|J'ai une liste avec 1 seule entrée, l'état actuel de la perturbation avec son numéro de version et sa dernière date de modification|
|J'ai modifié une perturbation|Je consulte la perturbation dans Navitia|Je ne vois que la dernière version|
|J'ai une perturbation|Je consulte la liste d'historique de ma perturbation|Je vois l'historique avec une section pagination|

## Team reviewers
